api/api_utils.py

```diff
@@ -8,44 +8,44 @@
 parentdirectory = os.path.dirname(currentdirectory)
 sys.path.insert(0, parentdirectory)
 
-from database import Function
+from database import DbOperation
 import utils
 
 
 def create(resourcename, payload={}):
```

Comment:
Using mutable dictionaries in function arguments are dangerous.
If you are not sure using None default argument with 
```
if payload = None:
  payload = {}
```
Is advised